ansible/roles/common/templates/boot.sh

```diff
@@ -43,4 +48,27 @@ do
   "$file"
 done
 
+# Mark first_boot_done and first_boot_failed only after the
```

Comment:
Meh, no need to leave the comment here. Move it to the commit message, so the intrepid code archeologist can find it, one year from now, when we've rewritten the file 3 times.

Reply:
I'm leaving the comment there for us to it to python later without breaking the tests in a non-reproducible way (say, some services are not started when `wait_first_boot` picks up the file).

When someone refactors that spaghetti, they'll kill the comments too.

Reply:
:+1: